include/deal.II/base/aligned_vector.h

```diff
@@ -542,7 +544,10 @@ AlignedVector<T>::reserve (const size_type size_alloc)
 
       // allocate and align along 64-byte boundaries (this is enough for all
       // levels of vectorization currently supported by deal.II)
-      T *new_data = static_cast<T *>(_mm_malloc (size_actual_allocate, 64));
+      T *new_data;
+      const int ierr = posix_memalign ((void **)&new_data, 64, size_actual_allocate);
+
+      AssertThrow (ierr == 0, ExcOutOfMemory());
 #else
```

Comment:
To be consistent with the two other spots, you can delete the `#if DEAL_II_COMPILER_VECTORIZATION_LEVEL > 0` part because we can do this unconditionally.